Mods/Pet_Default_v5_Dark.theme

```diff
@@ -1,20 +1,24 @@
 @description Default v5 Dark Theme
-@version 1.0
+@version 1.1
 @changelog
   - Updated to v5.22 (added preview)
+v1.1
+  - MIDI Editor Colormap reworked
+  - Changes in the ME Tracklist
+  - Separators added  
 @provides
   Default_5_Dark.ReaperThemeZip http://stash.reaper.fm/29513/Default_5_Dark.ReaperThemeZip
```

Comment:
I see three options:

1. What you suggested: creating a new .theme package would allow users to install the v1.0 and/or the v1.1 extended one as they wish
2. You could also bundle a "v5 Dark Compatibility" .ReaperThemeZip along. Users would install "v5 Dark" and get both variants in one shot since they're in fact the same theme (are they?)
3. Or just release an update and let users either not update or downgrade if too late (I think this is the worse solution)